internal/config/mount_config.go

```diff
@@ -35,9 +35,10 @@ const (
 	// The maximum multiple of seconds representable by time.Duration.
 	MaxSupportedTtlInSeconds int64 = int64(math.MaxInt64 / int64(time.Second))
 
-	// DefaultTypeCacheMaxEntries is the default vlaue of type-cache-max-entries.
-	// The value is set high at 1,048,476 (2^20) as it is not supposed to be set by users.
-	DefaultTypeCacheMaxEntries int = 2 << 20
+	// DefaultTypeCacheMaxSizeInMb is the default vlaue of type-cache max-size for every directory in MiBs.
```

Comment:
i think it's better to remove "In" to avoid confusion with the user flag.

Reply:
done